meta-resin-jethro/recipes-kernel/linux-firmware/linux-firmware_git.bbappend

```diff
@@ -1,5 +1,7 @@
 PACKAGES =+ "${PN}-iwlwifi-7265D-10"
-
 RDEPENDS_${PN}-iwlwifi-7265D-10 = "${PN}-iwlwifi-license"
-
 FILES_${PN}-iwlwifi-7265D-10 = "/lib/firmware/iwlwifi-7265D-10.ucode"
+
+PACKAGES =+ "${PN}-iwlwifi-8000C-13"
+RDEPENDS_${PN}-iwlwifi-8000C-13 = "${PN}-iwlwifi-license"
+FILES_${PN}-iwlwifi-8000C-13 = "/lib/firmware/iwlwifi-8000C-13.ucode"
```

Comment:
There is another commit which adds it to the packagegroup which is present on image.